### DISCUSSION TITLE
WebView2 Control support on .Net 7

### DISCUSSION THREAD

**Discussion by ketan-shah** [MicrosoftEdge/WebView2Feedback#4129] [Q&A]
Is there a plan to support WebView2 Control on .Net 7 and if so, when is that planned?
Thanks

**Comment by champnic** [comment]
The WebView2 control should work with .NET 7. Are you running into an error?

**Reply by ketan-shah** [reply]
It works in .Net 6 but not in .Net 7
I had to move my project and other dependencies to .Net 6 in order to use WebView2.
Also Microsoft Documentation does not mention of it being supported on .Net 7.
I don't recall the error it was throwing but I believe I did try all that I could but it would not work with .Net7

**Reply by champnic** [reply]
I've verified with a sample app on .NET 7 and the .NET 8 preview. Do you have some sample code you can share to demonstrate the failure? Or if you have more information on the issue and you believe it's due to a bug on WebView2 control, please open an issue in this repo. Thanks!
